requre/online_replacing.py

```diff
@@ -204,31 +222,34 @@ def _parse_and_replace_sys_modules(
     return original_module_items
 
 
-def _change_storage_file(storage_file, func, args):
+def _change_storage_file(
+    cassette: Cassette, func, args, storage_file: Optional[str] = None
+):
```

Comment:
Can we transform this to `cassette` method or maybe return the right value and set to `cassette` outside of this function?

 (I personally don't like changing the inputs inside function. It's harder to debug and test.)